code/modules/power/lighting.dm

```diff
@@ -61,9 +66,42 @@ var/global/list/light_type_cache = list()
 			to_chat(user, "It's wired.")
 		if(3)
 			to_chat(user, "The casing is closed.")
+	if(cell_connectors)
+		if(cell)
+			to_chat(user, "You see [cell] inside the casing.")
+		else
+			to_chat(user, "The casing has no power cell for backup power.")
+	else
+		to_chat(user, "<span class='danger'>This casing doesn't support power cells for backup power.</span>")
+
+/obj/machinery/light_construct/attack_hand(mob/user)

```

Comment:
Shouldn't this have a ..() somewhere in case of construction steps being handled on /obj/machinery, or are all the other steps done solely in attackby()?

Reply:
/obj/machinery/attack_hand(user) does some checks on whether user can use the machine.  I've now updated /obj/machinery/light_construct/attack_hand(user) to call it.